decidim-core/app/cells/decidim/tos_page/form.erb

```diff
@@ -1,19 +1,13 @@
-<div class="columns small-12">
-  <div class="card">
-    <div class="card__content">
-      <div class="card__header">
-        <h5 class="card__title text-center">
-          <%= t("form.legend", scope: "decidim.pages.terms_and_conditions") %>
-        </h5>
-      </div>
+<div class="rounded border border-gray p-4">
+  <div class="h5 text-center">
```

Comment:
Here you addressed the previous review's comment regarding incorrect heading level. However, I think this is now problematic for screen reader users as this form is particularly interesting for screen reader users. They are no different than normal users, they just want to quickly skip through all the long texts and "accept everything" automatically, so we should also understand their use cases.

When we take out the heading level from this heading, the screen reader users won't find this section now easily from the document. Many times they glance through the pages going through the links and headings on these pages, so we should assist them that this form actually contains something important for them.

I think a `<h2>` heading should work here in all cases. The page heading is `<h1>` and the page text can have any level headings. Then the final section for accepting the terms is important enough IMO that we can use `<h2>` here.

Another problem is that if we leave out the heading level completely from this element, semantically this section belongs to what ever happens to be the final heading of the terms of service.

E.g. if you take a look at the Barcelona instance's TOS page:
https://www.decidim.barcelona/pages/terms-and-conditions

On this page, this form would be now logically under the last section "F. Copyright, reuse of information and other rights on the website and its contents".

We can simply solve this by making this a `<h2>`, just wanted to point out some reasoning.